docs/src/examples/components/Label/Variations/LabelExampleColor.shorthand.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Label, ProviderConsumer, Grid } from '@stardust-ui/react'
+
+const LabelExampleColor = () => (
+  <Grid
```

Comment:
just thoughts aloud: would rather prefer them being aligned horizontally, as it is commonly done by other libraries for tag components - but yes, there is no good layout component for that yet

Reply:
Waiting for the Flex component then :)

Reply:
Always delete as much code as possible:
1. No Grid is necessary, especially since it is a very complex solution to a very simple problem (add a space).
1. Lodash is not necessary to Start Case the color names, just show the prop values (more helpful as well).
1. Only use top level parent components, not the directly exported children (Provider.Consumer vs ProviderConsumer)

![image](https://user-images.githubusercontent.com/5067638/51069927-65270800-15ed-11e9-8ee2-1b3a1e41bbd6.png)

```diff
import * as React from 'react'
-import * as _ from 'lodash'
-import { Label, ProviderConsumer, Grid } from '@stardust-ui/react'
+import { Label, Provider } from '@stardust-ui/react'

const LabelExampleColor = () => (
-  <Grid
-    columns="auto"
-    styles={{ justifyContent: 'left', justifyItems: 'left' }}
-    variables={{ gridGap: '10px' }}
-  >
    <ProviderConsumer
      render={({ siteVariables: { colorScheme } }) =>
        _.keys(colorScheme).map(color => (
-          <Label key={color} color={color} content={_.startCase(color)} />
+          <span>
+            <Label key={color} color={color} content={color} />{' '}
+          </span>
        ))
      }
    />
-  </Grid>
)
```